### DISCUSSION TITLE
被B站网友逮住了…

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#3085] [其他]
![Screenshot_20240819_170524](https://github.com/user-attachments/assets/e2fcc0d3-01c9-438b-bf8b-4d2be5107217)
发个帖子纪念一下

**Comment by jiaoaodedihuazhixiu** [comment]
遇见真警官了，能给个电子签名吗<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Reply by ZhuYuan-148** [reply]
刑侦特勤组，朱鸢。

**Comment by excellen114514** [comment]
朱鸢探长！我也想要电子签名！<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >
